tools/fab/frontend-install.py

```diff
@@ -1,15 +1,40 @@
-#!/usr/bin/python
+#! /usr/bin/env any_python
```

Comment:
I don't like that this is using a shabang the wouldn't exist if you didn't run frontend-install.sh. I'd rather it didn't have a shabang at all, and we forced the user to run `python frontend-install.py` or `python3 frontend-install.py`, depending on which python that had. Or leave it defaulted to `python` which is currently a good default for our main platforms.

Reply:
I mean it's either putting the `any_python` or some similar logic sprinkled through all the tools that use `frontend-install.*` or putting it here. I tend to lean towards reducing the spread of magic as much as possible, so that's why I went this route.

Reply:
But if there is a cleaner alternative, I'm all ears. This does feel gross but seemed the least gross of the things I could come up with.